Dockerfile

```diff
@@ -16,7 +16,10 @@ FROM quay.io/ceph/spdk:${NVMEOF_SPDK_VERSION:-NULL} AS base-gateway
 RUN \
     --mount=type=cache,target=/var/cache/dnf \
     --mount=type=cache,target=/var/lib/dnf \
-    dnf install -y python3-rados
+    dnf install -y python3-rados python3-pip
+RUN \
+    --mount=type=cache,target=/root/.cache/pip \
+    pip install -U pip setuptools
```

Comment:
why do we need pip in the gateway image?

Reply:
This is not the spdk file, it's the nvmeof one. We need pip as it used to install the tabulate and yaml packages.

Reply:
Reverted the change to Dockerfile